src/fetch/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- Rename package to `@adeira/fetch`
```

Comment:
Pls remove this line and just backup this file (see Eslint Config changelog for an example).